tests/main/res/config_hvdc_line_inverse.json

```diff
@@ -0,0 +1,7 @@
+{
+  "dfl-config": {
```

Comment:
This inverse test case should be private as it comes from a CGMES file. @marcochiaramello @rosiereflo  what do you think ?
I think we should remove it from the tests in the public repo and just do the inverse trick in the other case (we can inverse the lcc or the vsc). This way, in the same case we will have the normal configuration for one line and the inverse one for the other line.

Reply:
@PerninGuillaume Can you add as a test the last iidm that I sent to you (the one with one vsc and one lcc) and inverse one of the two lines (for example you can write converterStation1="VSCStation2" converterStation2="VSCStation1" instead of converterStation1="VSCStation1" converterStation2="VSCStation2")